examples/grocery_alf_ppo.gin

```diff
@@ -27,8 +27,8 @@ ActorDistributionNetwork.activation_fn=@tf.nn.tanh
 ActorDistributionNetwork.continuous_projection_net=@NormalProjectionNetwork
 NormalProjectionNetwork.init_means_output_factor=1e-10
 NormalProjectionNetwork.std_bias_initializer_value=0.0
-NormalProjectionNetwork.scale_distribution=True
-estimated_entropy.assume_reparametrization=True
+# NormalProjectionNetwork.scale_distribution=True
+# estimated_entropy.assume_reparametrization=True
```

Comment:
Is this used or not? If not, it should be removed. And ALF master now should support this now.

Reply:
Removed. I was meant to use this, but it seems current version is good enough as a simple example.